cloud_experts_tutorials/cloud-experts-gitops-with-argocd.adoc

```diff
@@ -0,0 +1,181 @@
++
+[source,terminal]
+----
+$ git clone https://github.com/rh-mobb/gitops-demo gitops
+----
+
+3. Export your local path to the GitHub files:
++
+[source,terminal]
+----
+$ export GITOPS_HOME="$(pwd)/gitops"
+$ cd $GITOPS_HOME
+----
+
+4. Log in to OpenShift by copying the login command from the OpenShift console:
++
+image::oc_login.png[screenshot of login]
+
+5. Enter the command in your terminal to authenticate with the OpenShift CLI (oc).
++
+.Sample output
++
+[source,terminal]
+----
+Logged into "https://<YOUR-INSTANCE>.openshiftapps.com:6443" as "<YOUR-ID>" using the token provided.
+----
+
+== Deploy the ArgoCD Project
+
+1. Create a new OpenShift project called *gitops* by running the following command:
++
+[source,terminal]
+----
+$ oc new-project gitops
+----
+
+2. Add **cluster-admin** rights to the `openshift-gitops-argocd-application-controller` service account in the **openshift-gitops** namespace by using the following command:
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user cluster-admin -z openshift-gitops-argocd-application-controller -n openshift-gitops
+----
+
+3. Retrieve the ArgoCD URL to log into your ArgoCD:
++
+[source,terminal]
+----
+$ argoURL=$(oc get route openshift-gitops-server -n openshift-gitops -o jsonpath='{.spec.host}{"\n"}')
+$ echo $argoURL
+----
+
+4. Retrieve ArgoCD Password:
++
+[source,terminal]
+----
+$ argoPass=$(oc get secret/openshift-gitops-cluster -n openshift-gitops -o jsonpath='{.data.admin\.password}' | base64 -d)
+$ echo $argoPass
+----
+
+5. In a web browser, navigate to the ArgoCD console using the `$argoURL` value:
++
+image::argo1.png[screenshot of argocd1]
+
+6. Log in with the user name **admin** and the password returned as `$argoPass`:
++
+image::argo2.png[screenshot of argocd2]
+
+7. Optional: if you want to use the command-line interface, log in to the CLI by running the following command:
++
+[source,terminal]
+----
+$ argocd login --insecure --grpc-web $argoURL  --username admin --password $argoPass
+----
+
+8. Use the `kubectl` CLI tool to apply the `bgd-app.yaml` file
++
+[source,terminal]
+----
+$ kubectl apply -f documentation/modules/ROOT/examples/bgd-app/bgd-app.yaml
+----
++
+The `bgd-app.yaml` file defines several things, including the repository's location for the `gitops-bgd-app` application:
++
+image::bgd-app-yaml.png[screenshot of bgd-app-yaml]
+
+9. Check the rollout running the following command:
++
+[source,terminal]
+----
+$ kubectl rollout status deploy/bgd -n bgd
+----
+
+10. Once the rollout is completed, get the route to the application by running the following command:
++
+[source,terminal]
+----
+$ oc get route bgd -n bgd -o jsonpath='{.spec.host}{"\n"}'
+----
+
+11. In your browser, paste the route to open the application:
++
+image::app_blue.png[screenshot of app_blue]
+
+12. Go back to your ArgoCD window and verify the configuration shows:
```

Comment:
```suggestion
. Go back to your Argo CD window and verify that the configuration shows:
```
I really don't like ending with "shows" here. Shows what? But it might be something to fix on the second round.